packages/cli/README.md

```diff
@@ -240,6 +240,28 @@ Run plugins, collect results and upload the report to the Code PushUp portal.
 
 Refer to the [Common Command Options](#common-command-options) for the list of available options.
 
+#### `history` command
+
+Usage:
+`code-pushup history`
+
+Description:
+Run plugins, collect results and upload the report to the Code PushUp portal for a specified amount of the repository history.
+
+Refer to the [Common Command Options](#common-command-options) for the list of available options.
+
+| Option                   | Type      | Default  | Description                                                      |
+| ------------------------ | --------- | -------- | ---------------------------------------------------------------- |
+| **`--targetBranch`**     | `string`  | 'main'   | Branch to crawl history.                                         |
+| **`--forceCleanStatus`** | `boolean` | `report` | If we reset the status to a clean git history forcefully or not. |
+| **`--maxCount`**         | `number`  | 5        | Number of steps in history.                                      |
+| **`--skipUploads`**      | `boolean` | `false`  | Upload created reports                                           |
+| **`--from`**             | `string`  | n/a      | Hash to start in history                                         |
+| **`--to`**               | `string`  | n/a      | Hash to end in history                                           |
```

Comment:
Wouldn't it be more user friendly to have a default value `HEAD` (current commit)?

Reply:
I commented on it here:
https://github.com/code-pushup/cli/blob/4f877217698a990e25e3cc83279d9e49f0fa548c/packages/core/src/lib/history.ts#L67-L105

And here:
https://github.com/code-pushup/cli/blob/6a7c7e681a5e97210997d7530997f49e34d767e8/packages/core/src/lib/history.ts#L113-L116

Also the tests would fail because of errors.